src/integration/blockchain/realunit/realunit-blockchain.service.ts

```diff
@@ -1,38 +1,138 @@
 import { Injectable, OnModuleInit } from '@nestjs/common';
 import { ModuleRef } from '@nestjs/core';
 import { Contract } from 'ethers';
+import {
+  AllowlistStatusDto,
+  BrokerbotBuyPriceDto,
+  BrokerbotInfoDto,
+  BrokerbotPriceDto,
+  BrokerbotSharesDto,
+} from 'src/integration/realunit/dto/realunit.dto';
 import { Blockchain } from '../shared/enums/blockchain.enum';
 import { EvmClient } from '../shared/evm/evm-client';
 import { EvmUtil } from '../shared/evm/evm.util';
 import { BlockchainRegistryService } from '../shared/services/blockchain-registry.service';
 
+// Contract addresses
+const BROKERBOT_ADDRESS = '0xCFF32C60B87296B8c0c12980De685bEd6Cb9dD6d';
+const REALU_TOKEN_ADDRESS = '0x553C7f9C780316FC1D34b8e14ac2465Ab22a090B';
+const ZCHF_ADDRESS = '0xb58e61c3098d85632df34eecfb899a1ed80921cb';
+
+// Contract ABIs
+const BROKERBOT_ABI = [
+  'function getPrice() public view returns (uint256)',
+  'function getBuyPrice(uint256 shares) public view returns (uint256)',
+  'function getShares(uint256 money) public view returns (uint256)',
+  'function settings() public view returns (uint256)',
+];
+
+const REALU_TOKEN_ABI = [
+  'function canReceiveFromAnyone(address account) public view returns (bool)',
+  'function isForbidden(address account) public view returns (bool)',
+  'function isPowerlisted(address account) public view returns (bool)',
+];
```

Comment:
Is that right? For me this change looks strange if I compare it to the `BROKER_BOT_ABI` before